line-sdk/src/test/java/com/linecorp/linesdk/auth/internal/BrowserAuthenticationApiTest.java

```diff
@@ -35,6 +34,9 @@
 import java.util.Collection;
 import java.util.List;
 
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
```

Comment:
I found these lines moved after rearranging imports. Suggest rearranging imports.

Reply:
it's the result after re-arrangement. :)

Reply:
okay..it seems like my format is different from SDK's.